lib/homework/bloc.dart

```diff
@@ -79,10 +80,10 @@ class HomeworkDownloader extends CollectionDownloader<Homework> {
   }
 }
 
-class SubmissionDownloader extends CollectionDownloader<Submission> {
+class _SubmissionDownloader extends CollectionDownloader<Submission> {
```

Comment:
It appears that there has been an API change and the `userId` is now called `studentId`. This needs to be changed in the `downloadAll` method and in `homework/data.dart`.